ansible_ai_connect/ai/api/model_client/grpc_client.py

```diff
@@ -83,9 +83,9 @@ def infer(self, request, model_input, model_id="", suggestion_id=None) -> Dict[s
 
     def self_test(self) -> HealthCheckSummary:
         url = (
-            f"{settings.ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL}://"
+            f"{settings.ANSIBLE_AI_MODEL_MESH_PROTOCOL}://"
```

Comment:
This change feels a little confusing to me, because the protocol for this client is grpc, but this value would need to be `http` for the healthcheck specifically. Should we rename this similar to `ANSIBLE_GRPC_HEALTHCHECK_PORT`, e.g. `ANSIBLE_GRPC_HEALTHCHECK_PROTOCOL`? Or leave the name as it was?